fvm/src/kernel/mod.rs

```diff
@@ -278,7 +278,7 @@ pub trait RandomnessOps {
     /// beacon from a given epoch.
     /// This randomness is not tied to any fork of the chain, and is unbiasable.
     fn get_randomness_from_beacon(&self, rand_epoch: ChainEpoch)
-        -> Result<[u8; RANDOMNESS_LENGTH]>;
+    -> Result<[u8; RANDOMNESS_LENGTH]>;
```

Comment:
yuk, this seems like the opposite of the introduction of indenting rules in other places!

Reply:
Yeah, I agree...